src/Evolution/Systems/Cce/OptionTags.hpp

```diff
@@ -122,6 +122,14 @@ struct H5Interpolator {
   using group = Cce;
 };
 
+struct H5IsBondiData {
+  using type = bool;
+  static constexpr OptionString help{
+      "true for boundary data in Bondi form, false for metric data."};
```

Comment:
Maybe elaborate a bit that the metric data is what was previously stored, but storing the Bondi data is much more memory efficient. Ultimately both need to be support.

Reply:
Do you mean in the code comment or the help text? I don't recall whether this help text is one of the cases where we have length limits.

Reply:
I was thinking the help string. We no longer have limits on help string lengths, or it's something like several hundred chars, but I think they're eliminated. We have a formatter now for the output